src/main/java/com/google/sps/servlets/EmailServlet.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet that sends emails to the author and followers of an answered question or a commented
+ * answer.
+ */
+@WebServlet("/email")
+public class EmailServlet extends HttpServlet {
+
+  /**
+   * Sends email push notifications when an question is answered or an answer is commented.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get type of notification to create from query string.
+    String typeOfNotification = request.getParameter("typeOfNotification");
+    // Get ID of modified element from query string and convert to int.
+    int modifiedElementId = Integer.parseInt(request.getParameter("modifiedElementId"));
+
+    // Define SQL access variables to avoid repetition.
+    String SQL_URL = Utility.SQL_CLOUD_URL;
+    String SQL_USER = Utility.SQL_CLOUD_USER;
+    String SQL_PASSWORDD = Utility.SQL_CLOUD_PASSWORD;
```

Comment:
Rather than reassigning local variables to this class, you can directly use the consts from the utility class. So wherever you need the URL, you can just call Utility.SQL_CLOUD_URL, etc.

Reply:
I had done it that way for readability's sake but maybe it isn't worth it the memory tradeoff.